### ISSUE TITLE
Create illustrations for merged object downloads

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/scpca-docs#218]
We need illustrations for SCE, AnnData, and CITE-seq. Here's the "pseudotree" from our meeting notes:

```
merged_object.{extension}
merged_summary_report.html
individual_reports/
└── individual_library_id/
        ├──library_id_qc.html
        └──library_id_cell_type.html
```

Before we start work, this issue should be updated to be more descriptive.

**Comment by allyhawkins** [comment]
@dvenprasad, as we discussed in slack, here are examples of the download trees. 

### SingleCellExperiment/R (single-cell, multiplexed, CITE)
For all projects with single-cell, multiplexed, and CITE-seq the download will be the same for `SingleCellExperiment` objects. 
```
SCPCP000000_merged.rds
SCPCP000000_summary_report.html
individual_reports/
└── SCPCL000000/
        ├──SCPCL000000_qc.html
        └──SCPCL000000_cell_type_report.html
```


### AnnData/Python (single-cell, multiplexed)
For all projects with single-cell and multiplexed, the download will be the same for `AnnData` objects. 
The only time this should be different is if a project has CITE-seq. 
```
SCPCP000000_merged.hdf5
SCPCP000000_summary_report.html
individual_reports/
└── SCPCL000000/
        ├──SCPCL000000_qc.html
        └──SCPCL000000_cell_type_report.html
```

### AnnData/Python (CITE)
For downloading AnnData of only projects that have CITE. 
```
SCPCP000000_merged_rna.hdf5
SCPCP000000_merged_adt.hdf5
SCPCP000000_summary_report.html
individual_reports/
└── SCPCL000000/
        ├──SCPCL000000_qc.html
        └──SCPCL000000_cell_type_report.html
```

Let me know if you have any questions or if I missed anything!

**Comment by dvenprasad** [comment]
@allyhawkins I want to clarify if the QC reports for merged objects are named `SCPCL000000_qc.html`

The QC reports in the current download images are labeled `SCPCL000000_qc_report.html`.  Do they need to change as well?

**Comment by allyhawkins** [comment]
> @allyhawkins I want to clarify if the QC reports for merged objects are named `SCPCL000000_qc.html`
> 
> The QC reports in the current download images are labeled `SCPCL000000_qc_report.html`. Do they need to change as well?

Good catch, that's correct! The file names should be `SCPCL000000_qc.html`.

**Comment by dvenprasad** [comment]
**SingleCellExperiment/R (single-cell, multiplexed, CITE)**
![merged-sc-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/8bd64b8f-1667-4256-aa28-9f8fd060c403)

**AnnData/Python (single-cell, multiplexed)**
![merged-anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/dafc6db6-4e0a-40ff-88d2-f8c9aa50ad76)

**AnnData/Python (CITE)**
![merged-anndata-cite-seq-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/6d4295ac-b94f-4768-ad8f-1dd876b044d0)


I will put the corrected (qc and celltype) download folder images in another ticket.  Let me know if these are good.

**Comment by sjspielman** [comment]
These look good to me, @dvenprasad! The only change I might actually make is renaming `_summary_report.html` -> `_merged_summary_report.html`, so it's super clear the report is only about the merged object, and not a summary of every library in the project. 

@allyhawkins can you have a look too and weigh in if you agree with me on 👆 or not?

**Comment by allyhawkins** [comment]
> These look good to me, @dvenprasad! The only change I might actually make is renaming `_summary_report.html` -> `_merged_summary_report.html`, so it's super clear the report is only about the merged object, and not a summary of every library in the project.
> 
> @allyhawkins can you have a look too and weigh in if you agree with me on 👆 or not?

I'm fine with that change. We just need to remember to change that in the workflow. 
Also @dvenprasad see my comment (https://github.com/AlexsLemonade/scpca-docs/issues/226#issuecomment-1885843813) about changing from `SCPCL000000_celltype_report.html` to `SCPCL000000_celltype-report.html`.

**Comment by dvenprasad** [comment]
Okay added `merged_summary_report.html` and corrected the cell type report to `celltype-report`


**SingleCellExperiment/R (single-cell, multiplexed, CITE)**
![merged-sc-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/f80f0181-54e1-4ca3-b9f5-8cf66ae263c4)

**AnnData/Python (single-cell, multiplexed)**
![merged-anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/aa4288f8-25ba-4331-a8c3-99988b09f7ae)

**AnnData/Python (CITE)**
![merged-anndata-cite-seq-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/c2c08037-360f-4b60-99e6-b683d18d3570)

**Comment by sjspielman** [comment]
wait! @dvenprasad @allyhawkins , did we mean `SCPCP000000_merged-summary-report.html`??

**Comment by allyhawkins** [comment]
These `-` and `_` will be the death of me. I think if we do it for one report we need to do it for the other. Sorry @dvenprasad!

**Comment by dvenprasad** [comment]
Alternate suggestion: if we are struggling to keep track of the `_` and the `-`, why don't we make it easy on ourselves and just pick one? The meaning does not change and it's only a formatting choice. 

It's likely that we'll have a similar conversation in the future. We can be kind to both our present and future selves (and others who might need to do this)

**Comment by allyhawkins** [comment]
> Alternate suggestion: if we are struggling to keep track of the _ and the -, why don't we make it easy on ourselves and just pick one? The meaning does not change and it's only a formatting choice.

@dvenprasad I totally agree that it would be easier to just have `_` like we do for all the other ones. But @jashapiro had requested that we use `-` for within chunk spacing. See https://github.com/AlexsLemonade/scpca-nf/pull/643#discussion_r1442208454. That's what prompted this change in the first place. But if it's easier I'm good with sticking with all `_`.

**Comment by jashapiro** [comment]
Jenny Bryan did a number on me.

**Comment by sjspielman** [comment]
Yeah, the idea is it's more of a naming convention to use _ to separate chunks of meaning, and - for spaces within each chunk. We haven't been very consistent with this, but I don't think it's a bad idea to be consistent with it forward.

**Comment by dvenprasad** [comment]
`finalv3_no_really_finalv4_actually_final.psd`


**SingleCellExperiment/R (single-cell, multiplexed, CITE)**
![merged-sc-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/d8797eee-b984-4605-b268-352fb0485124)

**AnnData/Python (single-cell, multiplexed)**
![merged-anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/a32bca6a-60b0-409d-93c4-ed7a159ccc3f)

**AnnData/Python (CITE)**
![merged-anndata-cite-seq-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/88fef943-36f7-4536-89e1-89bfb1e2aaa0)

**Comment by allyhawkins** [comment]
🎉

**Comment by sjspielman** [comment]
One more round I think 😬...

I think for **AnnData/Python (single-cell, multiplexed)**, we still need the `_rna` for the merged. So, `SCPCP000000_merged.hdf5` -> `SCPCP000000_merged_rna.hdf5`

https://github.com/AlexsLemonade/scpca-nf/blob/f08ce452f95f88e169217c114aef1c1321a3688c/merge.nf#L97-L105

**Comment by dvenprasad** [comment]
**AnnData/Python (single-cell, multiplexed)**
![merged-anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/1e214470-8624-4d9a-9377-43f7973a218f)

**Comment by sjspielman** [comment]
🐻‍❄️ i hear we like animal emojis around here

**Comment by dvenprasad** [comment]
All right here is merged object with single_cell_metadata.tsv and bulk files.

**SingleCellExperiment/R (single-cell, multiplexed, CITE)**
![merged-sc-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/1d2e34dc-99c7-47af-859a-7c11f20fc7e8)

**AnnData/Python (single-cell, multiplexed)**
![merged-anndata-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/70299c48-9516-4ce6-b63d-8d2d91407025)

**AnnData/Python (CITE)**
![merged-anndata-cite-seq-project-download-folder](https://github.com/AlexsLemonade/scpca-docs/assets/15315514/6aea9f9b-fcef-49bb-ac89-8b536b90225e)

**Comment by sjspielman** [comment]
@dvenprasad I think we've made it!!

In the interest of making sure, @allyhawkins can you look too?

**Comment by allyhawkins** [comment]
These look good to me! Thank you for dealing with all of our changes @dvenprasad

**Comment by sjspielman** [comment]
final.final closed by #256
